static/images/loader-black.svg

```diff
@@ -0,0 +1,49 @@
+<svg width="30px"  height="30px"  xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 100 100" preserveAspectRatio="xMidYMid" style="background: none;"><g transform="rotate(0 50 50)">
+  <rect x="47" y="15" rx="9.4" ry="3" width="6" height="20" fill="#000">
+    <animate attributeName="opacity" values="1;0" times="0;1" dur="1s" begin="-0.9166666666666666s" repeatCount="indefinite"></animate>
+  </rect>
+</g><g transform="rotate(30 50 50)">
+  <rect x="47" y="15" rx="9.4" ry="3" width="6" height="20" fill="#000">
+    <animate attributeName="opacity" values="1;0" times="0;1" dur="1s" begin="-0.8333333333333334s" repeatCount="indefinite"></animate>
+  </rect>
+</g><g transform="rotate(60 50 50)">
+  <rect x="47" y="15" rx="9.4" ry="3" width="6" height="20" fill="#000">
+    <animate attributeName="opacity" values="1;0" times="0;1" dur="1s" begin="-0.75s" repeatCount="indefinite"></animate>
+  </rect>
+</g><g transform="rotate(90 50 50)">
+  <rect x="47" y="15" rx="9.4" ry="3" width="6" height="20" fill="#000">
+    <animate attributeName="opacity" values="1;0" times="0;1" dur="1s" begin="-0.6666666666666666s" repeatCount="indefinite"></animate>
+  </rect>
+</g><g transform="rotate(120 50 50)">
+  <rect x="47" y="15" rx="9.4" ry="3" width="6" height="20" fill="#000">
+    <animate attributeName="opacity" values="1;0" times="0;1" dur="1s" begin="-0.5833333333333334s" repeatCount="indefinite"></animate>
+  </rect>
+</g><g transform="rotate(150 50 50)">
+  <rect x="47" y="15" rx="9.4" ry="3" width="6" height="20" fill="#000">
+    <animate attributeName="opacity" values="1;0" times="0;1" dur="1s" begin="-0.5s" repeatCount="indefinite"></animate>
+  </rect>
+</g><g transform="rotate(180 50 50)">
+  <rect x="47" y="15" rx="9.4" ry="3" width="6" height="20" fill="#000">
+    <animate attributeName="opacity" values="1;0" times="0;1" dur="1s" begin="-0.4166666666666667s" repeatCount="indefinite"></animate>
+  </rect>
+</g><g transform="rotate(210 50 50)">
+  <rect x="47" y="15" rx="9.4" ry="3" width="6" height="20" fill="#000">
+    <animate attributeName="opacity" values="1;0" times="0;1" dur="1s" begin="-0.3333333333333333s" repeatCount="indefinite"></animate>
+  </rect>
+</g><g transform="rotate(240 50 50)">
+  <rect x="47" y="15" rx="9.4" ry="3" width="6" height="20" fill="#000">
+    <animate attributeName="opacity" values="1;0" times="0;1" dur="1s" begin="-0.25s" repeatCount="indefinite"></animate>
+  </rect>
+</g><g transform="rotate(270 50 50)">
+  <rect x="47" y="15" rx="9.4" ry="3" width="6" height="20" fill="#000">
+    <animate attributeName="opacity" values="1;0" times="0;1" dur="1s" begin="-0.16666666666666666s" repeatCount="indefinite"></animate>
+  </rect>
+</g><g transform="rotate(300 50 50)">
```

Comment:
Could we use `static/assets/icons/` for this, to avoid additional assets?  I don't know how icon fonts work and thus if this idea makes sense.

Reply:
If not doing that, maybe we want to rename the existing one to `loader-white.svg` in a last commit, just for readability?

Reply:
I am also not sure whether we should move it to icons folder. Have added the commit to rename `loader.svg` to `loader-white.svg` for now.